build/git_test.go

```diff
@@ -131,6 +131,7 @@ func TestGetGitAttributes(t *testing.T) {
 
 func TestGetGitAttributesDirty(t *testing.T) {
 	setupTest(t)
+	t.Setenv("BUILDX_GIT_CHECK_DIRTY", "true")
```

Comment:
@crazy-max do we have documentation for this env-var somewhere? (or is that outside of this repository?)

Reply:
Oh we need it in https://docs.docker.com/build/building/env-vars/ yeah